integrations/event-handler/app.go

```diff
@@ -247,3 +253,89 @@ func (a *App) RegisterSession(ctx context.Context, e *TeleportEvent) {
 		log.Error("Registering session: ", err)
 	}
 }
+
+// Profile captures various Go pprof profiles and writes
+// them under the configured storage directory.
+func (a *App) Profile() {
```

Comment:
Two nits:
I think it should be a separate PR.
And we should probably move this to `integrations/lib/profile/` or `lib/profile/`